dockstore-webservice/src/main/java/io/dockstore/webservice/resources/DockerRepoResource.java

```diff
@@ -921,6 +950,39 @@ public void unstarEntry(@ApiParam(hidden = true) @Auth User user,
         return tool.getStarredUsers();
     }
 
+    @GET
```

Comment:
Let's move this to MetadataResource and get rid of the whitespace changes in the rest of the file

Reply:
Was considering this, but then the command that Kieran mentioned is `dockstore tool deps` which seemed more consistent when putting it in here.  Something else to consider is whether we would have `dockstore workflow deps` in the future.  If I do move it to MetadataResource, perhaps `dockstore deps` makes more sense.

Reply:
We're not likely to test with a different cwltool version for tools vs workflows. 
Move it out of DockerRepoResource, I'm fine with `dockstore deps`

Reply:
ok